### PR TITLE
Add text to icon-only buttons

### PR DESCRIPTION
**Pull request by s0nea**
~Add text to prefrences button. Add text to check- and x-icon on port-forwarding.~

Screenshot_2023-11-27_18-10-45
Screenshot_2023-11-27_18-13-19

~There are also a plus- and minus-icon on the Preferences -> Container Engine -> Allowed Images page. They're part of the string-list component and need to be adapted here: https://github.com/rancher/dashboard/blob/master/pkg/rancher-components/src/components/StringList/StringList.vue~

Fixes: https://github.com/rancher-sandbox/rancher-desktop/issues/4851

### CODE REVIEW

**Comment by jandubois** [comment]
> There are also a plus- and minus-icon on the Preferences -> Container Engine -> Allowed Images page. They're part of the string-list component

Can we discuss this in the next UI/UX meeting?

I'm undecided if I view these as "buttons" or as "controls" (like a scrollbar). If we consider them "buttons", what text would we use to replace the signs? Or would we consider "+" and "-" text (assuming they are not icons, but regular text, they should be accessible to screen readers).

**Comment by rak-phillip** [comment]
> I'm undecided if I view these as "buttons" or as "controls" (like a scrollbar). If we consider them "buttons", what text would we use to replace the signs? Or would we consider "+" and "-" text (assuming they are not icons, but regular text, they should be accessible to screen readers).

We should be able to make use of `aria-label` to provide content that is accessible to screen readers in a situation like this

https://www.w3.org/WAI/ARIA/apg/patterns/button/

File: pkg/rancher-desktop/assets/translations/en-us.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@
 generic:
   add: Add
   back: Back
+  ok: Ok
```
Comment (rak-phillip) [inline pkg/rancher-desktop/assets/translations/en-us.yaml:7]: I think we've on OK as the correct spelling

Screenshot 2023-11-28 at 8 52 39 AM

Comment (s0nea) [reply pkg/rancher-desktop/assets/translations/en-us.yaml:7]: Uhm, yes, of course! Thanks for the hint. I took a look at https://github.com/rancher-sandbox/rancher-desktop/pull/5281 and than I did it the wrong way :woman_facepalming:

**Review by rak-phillip — APPROVED** [review]
LGTM

**Pull request merged**